charts/kubeclarity/values.yaml

```diff
@@ -296,16 +296,16 @@ kubeclarity-sbom-db:
 #######################################################################################
 
 #######################################################################################
-## KubeClarity Postgres Values
-
+## KubeClarity Internal Postgres Values
+## Use kubeclarity-postgresql if you want this chart to deploy a PostgreSQL instance
 kubeclarity-postgresql:
   enabled: true
 
-  ## Specify posgtresql image
-  # image:
-    # registry: docker.io
-    # repository: bitnami/postgresql
-    # tag: 14.4.0-debian-11-r4
+  ## Specify posgtresql image used to check if postgresql DB is ready
```

Comment:
`used to check if postgresql DB is ready` - The command should be removed, it is not only for the init container, it is also used by the posgtresql chart following https://artifacthub.io/packages/helm/bitnami/postgresql#postgresql-common-parameters